modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -318,6 +361,15 @@ class Configuration(object):
                 'value': 'Bearer ' + self.access_token
             }
   {{/isBasicBearer}}
+  {{#isHttpSignature}}
+            '{{name}}':
```

Comment:
This is not assigning a value.
Can you change it to add the entry to the dict only if self.signing_info is not None?

Reply:
thanks. Side effect of sync from master.

Reply:
done.